### DISCUSSION TITLE
Current development progress?

### DISCUSSION THREAD

**Discussion by rourke** [monicahq/monica#6468] [General]
I'm considering using Monica very actively. I like it a lot so far, however I could see improvements here and there. I would like to invest hours of my time in Monica if I'm sure it's under active development. I prefer to stick to official releases, which is 3.7.0 at the moment, dating from February.

Can any of the developers say something about the current state of the app? Is a big version on its way? Is there any progress that can be tracked?

**Comment by caraar12345** [comment]
Looks like they're now building [Chandler](https://github.com/monicahq/chandler), a replacement for Monica 👀

**Comment by jcolinger3** [comment]
Is Monica dead?  Doesn't seem to be much conversation here.

**Comment by rourke** [comment]
I think it is. They shifted all focus to Chandler it seems. Last commit a few days ago.

**Reply by virtadpt** [reply]
Previous comment deleted because I didn't know about this.

**Reply by djaiss** [reply]
It's not. 

https://github.com/monicahq/monica/issues/6626

Chandler is simply Monica v2. Better, faster, etc-er...

**Comment by jcolinger3** [comment]
Are there some better locations that you know about for user dialogue?  These discussions are infrequent.

**Reply by djaiss** [reply]
The problem is having too many places for discussions. We, maintainers, won't be able to keep up with all these places. We do it on purpose - if we have one Slack, one Reddit, one Discord etc..., how can we find time to actually code :-) ?
